### PR TITLE
Fix Expo Module Template on iOS

### PR DESCRIPTION
**Pull request by sergeichestakov**
# Why

After I ran `expo generate-module` and tried to build the iOS client with my module as a dependency in the Podfile, I ran into a bunch of compilation errors. Turns out we seemed to have switched the library with the core native module functions from `EXCore` to `UMCore` and similarly changed all of the function names/classes from the format `EXExportedModule` to `UMExportedModule`.

# How

Rename the imported libraries, classes, and method names with the `UM` prefix rather than `EX`.

# Test Plan

After making these changes, my iOS client successfully compiled with the new unimodule included on my test iPhone X.

### CODE REVIEW

File: packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateModule.h [hunk 1]
```diff
@@ -1,7 +1,6 @@
 //  Copyright © 2018 650 Industries. All rights reserved.
+#import <UMCore/UMExportedModule.h>
```
Comment (sjchmiela) [inline packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateModule.h]:
```suggestion

#import <UMCore/UMExportedModule.h>
```

File: packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateViewManager.h [hunk 2]
```diff
@@ -1,8 +1,7 @@
 // Copyright 2016-present 650 Industries. All rights reserved.
+#import <UMCore/UMViewManager.h>
```
Comment (sjchmiela) [inline packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateViewManager.h]:
```suggestion

#import <UMCore/UMViewManager.h>
```

**Review by sjchmiela — APPROVED** [review]
I think we follow this convention of (comment, newline, imports) in most of the codebase, so let's stick to it here too. Apart from that LGTM! 👍

File: packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateViewManager.h [hunk 3]
```diff
@@ -1,8 +1,8 @@
 // Copyright 2016-present 650 Industries. All rights reserved.
```
Comment (EvanBacon) [inline packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateViewManager.h]: update this

File: packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateView.h [hunk 4]
```diff
@@ -1,9 +1,9 @@
 // Copyright 2016-present 650 Industries. All rights reserved.
```
Comment (EvanBacon) [inline packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateView.h]: same

File: packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateModule.h [hunk 5]
```diff
@@ -1,7 +1,7 @@
 //  Copyright © 2018 650 Industries. All rights reserved.
 
-#import <EXCore/EXExportedModule.h>
-#import <EXCore/EXModuleRegistryConsumer.h>
+#import <UMCore/UMExportedModule.h>
+#import <UMCore/UMModuleRegistryConsumer.h>
 
-@interface EXModuleTemplateModule : EXExportedModule <EXModuleRegistryConsumer>
+@interface EXModuleTemplateModule : UMExportedModule <UMModuleRegistryConsumer>
```
Comment (EvanBacon) [inline packages/expo-module-template/ios/EXModuleTemplate/EXModuleTemplateModule.h]: No module, thats Android

```suggestion
@interface EXModuleTemplate : UMExportedModule <UMModuleRegistryConsumer>
```

**Pull request merged**